week_1/Project/youtube-fe/styles/header.css

```diff
@@ -0,0 +1,123 @@
+.header {
+    height: 55px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    background-color: #fff;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    z-index: 100;
```

Comment:
@codal-kcolaco what is use of z-index?

Reply:
@abhi-codal z-index is used to determine stacking order of an element. It takes an integer value. Element with higher z-index will be stacked above elements with lower z-index.